$OpenBSD: patch-acinclude_m4,v 1.2 2017/11/05 15:12:56 jca Exp $

Zero out the memory before testing for overwrite bug.

Fixed in unstable, not in the gmp-6.1 branch:

  https://gmplib.org/repo/gmp/rev/2ce5c60f5372

Index: acinclude.m4
--- acinclude.m4.orig
+++ acinclude.m4
@@ -741,7 +741,7 @@ main ()
   long i;
   for (i = 0; i < 88 + 1; i++)
     a[i] = ~0L;
-  r = malloc (10000 * sizeof (unsigned long));
+  r = calloc (10000, sizeof (unsigned long));
   r2 = r;
   for (i = 0; i < 528; i += 23)
     {
